### DISCUSSION TITLE
Update function Moonraker / Fluidd / Mainsail

### DISCUSSION THREAD

**Discussion by Abatec75** [Guilouz/Creality-Helper-Script-Wiki#503] [General]
Hi there. I'm new here. I got my K1 Max a few days ago and I installed Guilouz helper script. The installations went flawless and all is working very fine. At this point - Thank you very much for your efforts Guilouz!

I understood, that Klipper itself is coustomized by creality and cannot be updated by us. But it didn't get what the situations are for Moonraker, Fluidd and Mainsail. I know they could be updated, but what is the right way to do it. Concerning Fluidd I tried to remove and then install it again with the helper script and yes that did the trick, I got the newest version. ...but what would be the better way, is there a update function I couldn't find?

Many thanks!

**Comment by Guilouz** [comment]
When you install Moonraker, Fluidd or Mainsail from script it's already official builds and for update them in future, just use update manager on Web interface.

**Comment by Abatec75** [comment]
Many many thanks Guilouz!

At the beginning I felt a little bit embarrassed or even blind, because I was not able to find any update functions on the web interface. But after having read the moonraker documentation, I realized that I have to define some "update manager" configs in moonraker.conf and boom - now the update manager is visible in fluidd and mainsail.

I learned something, many thanks again :-)
